core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -165,10 +164,9 @@ private Runnable commit(final Substream winningSubstream) {
       // subtract the share of this RPC from channelBufferUsed.
       channelBufferUsed.addAndGet(-perRpcBufferUsed);
 
+      final boolean wasCancelled = (scheduledRetry != null) ? scheduledRetry.isCancelled() : false;
       final Future<?> retryFuture;
       if (scheduledRetry != null) {
-        // TODO(b/145386688): This access should be guarded by 'this.scheduledRetry.lock'; instead
```

Comment:
Why remove these? These look disabled by the `@SuppressWarnings("GuardedBy")`

Reply:
The way that the comments were done was intrusive and ugly.  Changed them all to be on the `SuppressWarnings` annotation.